code/modules/projectiles/guns/energy/tesla.dm

```diff
@@ -0,0 +1,91 @@
+#define MIN_TO_FIRE 1000000
```

Comment:
just use the define MEGAWATT or define this as
`#define MIN_TO_FIRE 1 MEGAWATT`